### ISSUE TITLE
Single Fastfile for multiple projects

### ISSUE THREAD

**Issue by nikitaame** [fastlane/fastlane#7692]
Hi Team!

Awesome using Fastlane, thanks for making and maintaining it. I was trying to figure out a way to have a single Fastfile used for building numerous projects. I was planing on having a configuration file for each project and the ability to run lanes with an option that specifies which project to run the lane on. Would it be possible to have the Fastfile in a parent directory of the multiple projects?

Thanks!

**Comment by PetrosIchor** [comment]
You're question is a little vague as to design of your projects and expectations so it's hard to answer that question. 

From my experience the answer is yes.  I'm using fastlane with two platform focused products that cover about 16 apps (and growing). The platforms are using different technology, code bases, and targets different configurations for ios/android.  With a little tweaking of fastlane, I am able to accomplish all my build and deployment needs. 

While standing up fastlane for my organization, I went through a couple different designs for accomplishing these builds.  Found one that worked for my needs and it's used in production. 

My best advice is to heavily read through the documentation and examples.  When you hit a wall, search the issues here on github. 99% of the time someone's already asked the question and a solution is in the comments.  It's how I manage a more customized implementation of fastlane.
